_posts/0000-01-02-melimelo-git.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
It is a truth universally acknowledged that a single man in possession of a large fortune must be in want of a wife.-Jane Austen